src/set_theory/schroeder_bernstein.lean

```diff
@@ -2,12 +2,17 @@
 Copyright (c) 2017 Johannes Hölzl. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johannes Hölzl, Mario Carneiro
-
-The Schröder-Bernstein theorem, and well ordering of cardinals.
 -/
 import order.fixed_points
 import order.zorn
 
+/-!
+# Schröder-Bernstein theorem, well-ordering of cardinals
+
+This file proves (nonconstructively) the Schröder-Bernstein theorem (see `schroeder_bernstein`), the
```

Comment:
maybe write as a todo that it can be done constructively? (if I remember the proof right)

Reply:
also, probably worth documenting exactly what it is either here or in schroeder-bernstein